src/transform/plugins/images/index.ts

```diff
@@ -92,6 +92,52 @@ type Opts = SVGOpts & ImageOpts;
 const index: MarkdownItPluginCb<Opts> = (md, opts) => {
     md.assets = [];
 
+    md.inline.ruler.after('image', 'image_caption', (state, silent) => {
```

Comment:
Looks like this is yet another attribute parser.

Can we use already implemented instead of this?

There was attached [markdown-it-attrs](https://github.com/arve0/markdown-it-attrs/blob/master/index.js#L97) plugin.

So at this time (when we process something after image) image token should already contain target parsed attribute.

Or we can attach parser in next way:
```
md.core.ruler.after('curly_attributes', 'image_caption', imageCaption);
```

Reply:
Omg, that's what I needed, it could save 2 days if I knew about it. Updated the implementation using [markdown-it-attrs](https://github.com/arve0/markdown-it-attrs/blob/master/index.js#L97) plugin